lawnchair/src/app/lawnchair/search/data/SearchDataInfo.kt

```diff
@@ -0,0 +1,104 @@
+package app.lawnchair.search.data
+
+import app.lawnchair.search.data.MimeTypes.MIME_APP
+import app.lawnchair.search.data.MimeTypes.MIME_AUDIO
+import app.lawnchair.search.data.MimeTypes.MIME_CSV
+import app.lawnchair.search.data.MimeTypes.MIME_EXCEL
+import app.lawnchair.search.data.MimeTypes.MIME_IMAGES
+import app.lawnchair.search.data.MimeTypes.MIME_PDF
+import app.lawnchair.search.data.MimeTypes.MIME_PPT1
+import app.lawnchair.search.data.MimeTypes.MIME_PPT2
+import app.lawnchair.search.data.MimeTypes.MIME_SRT
+import app.lawnchair.search.data.MimeTypes.MIME_TEXT
+import app.lawnchair.search.data.MimeTypes.MIME_VIDEO
+import app.lawnchair.search.data.MimeTypes.MIME_WORD
+import app.lawnchair.search.data.MimeTypes.MIME_ZIP
+import com.android.launcher3.R
+
+object MimeTypes {
+    const val MIME_APP = "application/vnd.android.package-archive"
+    const val MIME_AUDIO = "audio/"
+    const val MIME_CSV = "text/comma-separated-values"
+    const val MIME_EXCEL = "application/vnd.ms-excel"
+    const val MIME_IMAGES = "image/"
+    const val MIME_PDF = "application/pdf"
+    const val MIME_PPT1 = "application/vnd.ms-powerpoint"
+    const val MIME_PPT2 = "application/vnd.openxmlformats-officedocument.presentationml.presentation"
+    const val MIME_SRT = "application/x-subrip"
+    const val MIME_TEXT = "text/"
+    const val MIME_VIDEO = "video/"
+    const val MIME_WORD = "application/msword"
+    const val MIME_ZIP = "application/zip"
+}
```

Comment:
I can intro a more complete mime types mapping in a separate PR.

Reply:
yeah that will be great

Reply:
We can use the mapping from #3790 now.

Reply:
done can you check